### PR TITLE
Bookmarking

### PR DESCRIPTION
**Pull request by AnnekeSchuette**
I added a bookmark function to my WineCard component including unit tests.
Thanks in advance for your feedback and improvement suggestions! 💣🚀🙏🏻

## Value statement

As a **user** 
I **need a bookmarking function on wine cards**
so that I can  ** add** them to a list of saved wines to quickly find them later on

## Acceptance criteria
- every WineCard has a Bookmark Icon 
- the bookmark-icon (heart) has two visual states: active / inactive (default)
- the state toggles on click
- bookmarked wines are **saved** into an array _savedWines_
- de-bookmarked Wines are **removed** from the array _savedWines_

### CODE REVIEW

**Review by VictoriaWika — COMMENTED** [review]
Nice work! 👍🏻 I left some small comments

File: client/src/components/App/App.js [hunk 1]
```diff
@@ -3,13 +3,17 @@ import Header from 'components/Header/Header'
 import { recommendedWines } from 'data/wine_recs_mixed_small.json'
 import WineListing from 'components/WineListing/WineListing'
 import quarterCircle from 'assets/quarterCircle.svg'
+import { useSaveWine } from 'lib/customHooks'
 
 export default function App() {
+  const [savedWines, { handleSaveWine }] = useSaveWine()
+  console.log(savedWines)
```
[the remark this answers is not in the document]
Comment (AnnekeSchuette) [reply client/src/components/App/App.js:10]: 😅

File: client/src/components/WineCard/WineCard.js [hunk 2]
```diff
@@ -30,12 +35,19 @@ export default function WineCard({
 
   return (
     <CardContent>
+      <h3>{title}</h3>
+      <BookmarkButton
+        onClick={() => handleBookmarkClick({ id, title })}
+        isActive={toggleState}
+        aria-label="Bookmark"
```
[the remark this answers is not in the document]
Comment (AnnekeSchuette) [reply client/src/components/WineCard/WineCard.js:42]: Thanks a lot for this helpful advice! I added the aria-label for testing purposes, but didn't really consider the value for the user.

**Comment by AnnekeSchuette** [comment]
> Nice work! 👍🏻 I left some small comments

@VictoriaWika Thanks a lot for your code review! I've made a few improvements based on your comments, fixed storybook and added a new test. It would be great, if you could approve the changes regarding your annotations. Merci! 🙏🏻

https://github.com/AnnekeSchuette/capstone-project/compare/6e708fbf75e3b2af5f6d6e8063f71c5c9f050686...0525124eff2af2873da312f25fd66fd5f106a791

**Comment by VictoriaWika** [comment]
Great Job! Everything looks good! 👍🏻

**Pull request merged**